QUARKUS-713.md

```diff
@@ -0,0 +1,46 @@
+# QUARKUS-713 Tech preview support for the OpenShift Client
+
+This document describes a high-level testing overview of [QUARKUS-713 Tech preview support for the OpenShift Client](https://issues.redhat.com/browse/QUARKUS-713).
```

Comment:
```suggestion
JIRA link: https://issues.redhat.com/browse/QUARKUS-713
```
To follow the common structure in all test plans

Reply:
Well, I like the @mjurc brief description.

Reply:
I'm gonna keep this one, a brief description never hurt anything :)